### ISSUE TITLE
New voicepack: Wheatley + PotatOS

### ISSUE THREAD

**Issue by somedoodest** [BEEmod/BEE2-items#3339]
A voice pack where Wheatley and PotatOS interact. (With some coding magic, this would give the player a PotatOS portal gun.) This is my solution of how to implement PotatOS, to avoid the problem of having her talk to theregular GLaDOS voice lines. (I might update this to include other voices with PotatOS in them, such as a "GLaDOS + PotatOS" or a "Cave + PotatOS".)
Potential voice lines:
- Before Entry:
Line trigger: Music is "Machiavellian Bach"
PotatOS: "Oh... Now he's playing classical music..."
- Entry:
1. Line trigger: Generic
PotatOS: "I think we're in trouble."
Wheatley: "Designed this test myself. It's a little bit difficult."
2. Line trigger: Generic
Wheatley: "It's alright! Everything's good. I just invented some more tests!"
PotatOS: "This is one of MY tests!"
Wheatley: "Uh, false. I'll go with false."
3. Line trigger: Turrets present
Wheatley: "Anyway, new turrets. Not defective. Ace of fours. The best hand. Unbeatable, I imagine."
PotatOS: "I'm sure we'll be fine."
- Mid:
1. Line trigger: Picked up Portal Gun
PotatOS: "Hey, moron."
Wheatley: "Oh. Hello."
PotatOS: "Alright, paradox time... THIS. STATEMENT. IS. FALSE. Don't think about it don't think about it don't think about it.."
Wheatley: "Um... True."
PotatOS: "It's a PARADOX! There IS no answer!"
Wheatley: "Uh, false. I'll go with false."
2. Line trigger: Pick up FrankenTurret
Wheatley: "For god's sake, you're BOXES with LEGS! It is literally your only purpose! Walking onto buttons! How can you not do the one thing you were designed for?"
PotatOS: "And...?"
- Exit: 
1. Line trigger: Picked up Portal Gun
PotatOS: "Alright. My paradox idea didn't work. AND it almost killed me. Luckily, by the looks of things, he knows as much about test building as he does about logical contradictions."
2. Line trigger: Pedestal Button present
Wheatley: "Ohh yes. Well done."
PotatOS: "Thanks! All we had to do is pull that lever."
Wheatley: "What? Well no, you pressed the b- AAAAAAAAAAAAAAH"
PotatOS: "Hehehehe... I know we're in a lot of trouble and probably about to die, but that was worth it."
3. Line trigger: Generic
Wheatley: "Are you... are you sure you're solving these correctly? Yes, you 'solved' it, but I'm wondering if there are a number of ways to solve them and you're picking all the worst ways."
PotatOS: "I think he's getting desperate."
Wheatley: "What was that?"
PotatOS: "Nothing. Nothing."
4. Line trigger: BTS, Infinite Pedestal Button present (if these can be detected)
Wheatley: "Ohh yes. Well done."
PotatOS: "Thanks! All we had to do is pull that le- [bzzpt]"

**Comment by somedoodest** [comment]
I thought i posted this in items. Whoops.

**Comment by Some-Username-Available** [comment]
I was thinking a PotatOS portal gun holder thing. So when picked up, PotatOS says the "Because I'm a potato' line.

**Comment by Naleksuh** [comment]
Classical music one won't work as music is entirely seperate from the voicelines

**Comment by TeamSpen210** [comment]
That's the point, if you choose that base track it'll do that line.

**Comment by Naleksuh** [comment]
Fine, but only if its also possible to _not_ have the voiceline play

**Comment by Luke18033** [comment]
Of course, you'd just unckeck that specific line in the config window.

**Comment by KaziPlays** [comment]
When will this be added?

**Comment by Luke18033** [comment]
Probably alongside Wheatley style, if that's ever added. Don't count on it happening anytime soon.

**Comment by Naleksuh** [comment]
Just got a notification from this. I don't remember commenting here before but sure this is cool
